pkg/utils/resources/resources.go

```diff
@@ -65,3 +74,24 @@ func Quantity(value string) *resource.Quantity {
 func IsZero(r resource.Quantity) bool {
 	return r.IsZero()
 }
+
+func Cmp(lhs resource.Quantity, rhs resource.Quantity) int {
```

Comment:
I realized that this helper is now only used in Fits, so we could potentially remove this.

Reply:
Checked and the scheduler uses it too: 

```
func byCPUAndMemoryDescending(pods []*v1.Pod) func(i int, j int) bool {
	return func(i, j int) bool {
		lhs := resources.RequestsForPods(pods[i])
		rhs := resources.RequestsForPods(pods[j])

		cpuCmp := resources.Cmp(lhs[v1.ResourceCPU], rhs[v1.ResourceCPU])
		if cpuCmp < 0 {
			// LHS has less CPU, so it should be sorted after
			return false
		} else if cpuCmp > 0 {
			return true
		}
		memCmp := resources.Cmp(lhs[v1.ResourceMemory], rhs[v1.ResourceMemory])

		if memCmp < 0 {
			return false
		} else if memCmp > 0 {
			return true
		}
		return false
	}
}
```